services/bls_aggregation/blsagg_test.go

```diff
@@ -1009,8 +1009,10 @@ func TestBlsAgg(t *testing.T) {
 			taskResponseDigest2, err := hashFunction(taskResponse2)
 			require.Nil(t, err)
 			blsSigOp2 := testOperator2BlsKeypair.SignMessage(taskResponseDigest2)
-			ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+			ctx, cancel := context.WithTimeout(context.Background(), 1*time.Nanosecond)
 			defer cancel()
+			time.Sleep(1 * time.Millisecond)
```

Comment:
Let's just use a duration of 0.
```suggestion
			ctx, cancel := context.WithTimeout(context.Background(), 0)
			defer cancel()
```

Reply:
Done in [797861f](https://github.com/Layr-Labs/eigensdk-go/pull/585/commits/797861fb63fd2c0666a2bbf68ddf93e72274601e)